combat-trainer.lic

```diff
@@ -1784,7 +1787,11 @@ class AttackProcess
 
       if (game_state.backstab? || game_state.ambush?) && hiding?
         if (game_state.backstab? && game_state.no_stab_current_mob) || (game_state.ambush? && !game_state.backstab?)
-          command += ' back'
+          if @ambush_location
+            command += " #{@ambush_location}"
```

Comment:
I think you can simplify this to `command += @ambush_location`

Reply:
Though if you default `@ambush_location` to `'back'`, you can get rid of this entire conditional

Reply:
Very true, getting rid of the conditional will break people who haven't restarted dependency to get the updated base though. Should I leave the conditional in for a few days then back it out when its reasonable to expect everyone has updated, or just drop it?

Reply:
I'd leave it in for a few days, with another PR to remove it